news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,211 @@
+
+United States entered this tournament with almost the same roster with swapping Kisetsunotori to Elux being the only roster change. Although they've fallen to the losers bracket after playing a match against South Korea, United States showed their determination to achieve something worthy. In the end, they've claimed themselfes as grand finalists for the second time in this world cup.
+
+With the recent change of swapping from Group Stages to Qualifiers format, most of the players and viewers didn't expect to see China on a very top despite failing their previous run placing 8th-7th place. Their expectations worth it in the end as they instantly gained their glory reaching winners finals bracket without losing a single match until meeting with South Korea and untimately to United States. Even with a long year absense, China showed that even being absent for an entire year they can still make a good fight with top teams.
+
+There's still plenty of tournaments running though even after the conclusion of CWC! The digit tournaments, 4 Digit Catch World Cup 2022 and 3 Digit Catch World Cup 2022, are up and running and a second iteration of Catch French Dual Cup 2 currently in the registered phase. There's nothing to talk about these tournaments yet but maybe with the new osu!monthly release we can fully recap what was going on with conclusions and stuff!
+
+### ![](/wiki/shared/mode/taiko.png) osu!taiko
+ #### As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, Taiko takes it upon themself to rival even that of *The Roundtable* with their very own annual entry of **[Expert Global Taiko Showdown 2022](https://osu.ppy.sh/wiki/en/Tournaments/GTS/EGTS_2022)**! 
+
+As the name implies, the showdown is to take place between those whose dreams consist of taiko drumming, kiai slider velocity shenanigans, dons, katsus, and if we even dare-say, taikonator 3000! This is a tournament that showcases those whose passion for Taiko is not just exemplary in their gameplay but in every other way. 
+
+---
+
+## Top scores of the Month!
+### ![](/wiki/shared/mode/osu.png) osu! 
+#### As if the impossible could not be done only once, but twice!? 
+[[Karcher]](https://osu.ppy.sh/users/11443437) is an absolute madman! Achieving an insane Full Combo on [*Yomi yori Kikoyu, Koukoku no Tou to Honoo no Shoujo*](https://osu.ppy.sh/beatmapsets/461744#fruits/1031991) to ever hit the ranked section. For just over 8 minutes of gameplay at a staggering 220 BPM it is no small feat, and thus, should be showed off for osu!’s monthly pick! 
```

Comment:
```suggestion

[\[Karcher\]](https://osu.ppy.sh/users/11443437) is an absolute madman! Achieving an insane Full Combo on [*Yomi yori Kikoyu, Koukoku no Tou to Honoo no Shoujo*](https://osu.ppy.sh/beatmapsets/461744#fruits/1031991) to ever hit the ranked section. For just over 8 minutes of gameplay at a staggering 220 BPM it is no small feat, and thus, should be showed off for osu!’s monthly pick!
```
Not sure about formatting here, but in wiki articles square brackets (`[` and `]`) need to be escaped, so they aren't interpreted by markdown.

Reply:
Wasn't sure about it, thanks for confirming!